runtime/connectors/connectors.go

```diff
@@ -79,7 +79,19 @@ func (ps PropertySchema) ValidateType(val any) bool {
 type Env struct {
 	RepoDriver string
 	RepoDSN    string
-	Variables  map[string]string
+	// user provided env variables
+	// following env variables are used in connectors :
+	// use_host_credentials - disable/enable use of host credentials :
```

Comment:
This setting shouldn't be configurable by the user (like `Variables` is), but by the deployer (in cloud, we want to fix force it to be false). So let's add it as a separate field (`AllowHostCredentials bool`).

Ideally, it should be a config option (`ConnectorsAllowHostCredentials`) in `runtime/runtime.go` in the `Options` struct. But if it's hard to propagate the value from there to the `connectors.Env`, we can parse it from `Variables` when creating the `connectors.Env` for now (and remember to make it safe when we refactor the reconcile).

Reply:
And to clarify, I think "allow" is the behavior we want, not "use" – it's a question of whether to fallback to host credentials if no overrides are provided (so we don't require active user decision).